data_model/src/events/data/events.rs

```diff
@@ -140,6 +142,27 @@ mod asset {
         pub asset_id: AssetId,
         pub amount: AssetValue,
     }
+
+    /// [`Self`] represents updated total asset quantity.
+    #[derive(
+        Clone,
+        PartialEq,
+        Eq,
+        PartialOrd,
+        Ord,
+        Hash,
+        Debug,
+        Decode,
+        Encode,
+        Deserialize,
+        Serialize,
+        IntoSchema,
+    )]
+    #[allow(missing_docs)]
+    pub struct AssetDefinitionTotalQuantityChanged {
```

Comment:
do we need this event? it should be implied by events emitted when asset is minted/burned

Reply:
Recommend keeping it.